### PR TITLE
[issue_51_version] adds -v/--version option

### PR DESCRIPTION
**Pull request by adam-tokarski**
### What's Changed

Simple PR that 
* provides possibility to print current `reviews` version
* solves installation in Docker issues by adding `libffi-dev` and `make` packages
* solves `pylint` issue when run from within a Docker
* adds minor optimization for building Docker container

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -9,3 +9,9 @@
 !.pylintrc
 !requirements.txt
 !requirements_dev.txt
+
+.dependabot
```
Comment (apoclyps) [inline .dockerignore:13]: The docker ignores whitelist should ignore all files that are not explicitly included in the docker ignore. The following snippet should be included at the top of this file (it is missing). 

```yaml
# Ignore everything
*
```

If you add the above snippet to the top of the `.dockerignore` file, you can remove the below files as they won't be copied as they are excluded from the whitelist.

see https://kevinpollet.dev/posts/how-to-use-your-dockerignore-as-a-whitelist/ for more on docker whitelists.

**Comment by apoclyps** [comment]
This is great 👍  

Thank you @adam-tokarski for submitting this pull request ❤️ . 

I only have 1 piece of feedback for you; afterward, I'm happy to approve and merge this pull request into `reviews`.

File: .dockerignore (hunk 1 above)
Comment (adam-tokarski) [reply .dockerignore]: `.dockerignore` is updated. The thing is that currently this file is ignored because of `volume` in `docker-compose`.

**Comment by adam-tokarski** [comment]
@apoclyps please take another look.

At this moment that `.dockerignore` is ignored, because of binded `volume`, but I think it's alright as it allows to work with that container in sensible way. When, in one day, someone will remove that volume configuration, the `.dockerignore` will be ready.

**Pull request merged**